### PR TITLE
On branch dev-amd-detach

### PR DESCRIPTION
**Pull request by youngyoonii**
Changes to be committed:
	modified:   mdistiller/distillers/AMD_recon.py
	modified:   mdistiller/engine/cfg.py
	
- Add on/off for the distillation teacher feature detach

### CODE REVIEW

File: mdistiller/engine/cfg.py [hunk 1]
```diff
@@ -83,6 +83,7 @@ def dump_cfg(cfg, show: bool=False):
 CFG.AMD.M_LAYERS = [5]  # manipulating layers
 CFG.AMD.ALIGN_TYPE = 'mse'  # 'cosine', 'mse', 'both'
 CFG.AMD.INPUT_SIZE = [224, 224]
+CFG.AMD.FEAT_DETACH = False
```
Comment (nda111) [inline mdistiller/engine/cfg.py]: `CFG.AMD.FEAT_DETACH`보다는 `CFG.AMD.LOSS.DETACH_REFINER` 뭐 이런 이름이 낫지않겠습니까

File: mdistiller/distillers/AMD_recon.py [hunk 2]
```diff
@@ -94,15 +96,16 @@ def forward_train(self, image, target, **kwargs):
                     ignore_outliers = not self.af_artifact_norm
                     reconstructor = self.reconstructor_dict[f"recon_mha_{m_l:03d}"]
                     (recon_f_t, _) , outlier_mask, _ = reconstructor(f_t, outlier_mask, ignore_outliers=ignore_outliers)
+                    distill_f_t = recon_f_t.clone().detach() if self.feature_detach else recon_f_t.clone()
```
Comment (nda111) [inline mdistiller/distillers/AMD_recon.py:99]: `clone()`한 이유는 loss 둘 중에 하나만 detach하려고 그러는 건가요?

**Review by nda111 — CHANGES_REQUESTED** [review]
하나만 수정해주시면 될 것 같습니다.

File: mdistiller/engine/cfg.py (hunk 1 above)
Comment (youngyoonii) [reply mdistiller/engine/cfg.py]: 확인했습니다~

File: mdistiller/distillers/AMD_recon.py (hunk 2 above)
Comment (youngyoonii) [reply mdistiller/distillers/AMD_recon.py:99]: 네네 뒤에 recon loss에서도 recon_f_t 를 써야해서 clone으로 썼습니다

**Review by nda111 — APPROVED** [review]
수정 감사합니다.
